src/storage/storage_common.h

```diff
@@ -179,7 +179,6 @@ typedef enum
   PAGE_HEAP,			/* heap page */
   PAGE_VOLHEADER,		/* volume header page */
   PAGE_VOLBITMAP,		/* volume bitmap page */
-  PAGE_XASL,			/* xasl stream page */
```

Comment:
@ribram 
Do we read the existing database volumes with the change?
Otherwise, I'd like to propose touching disk compatibility number.